Dart/thirdPartySrc/analysisServer/com/google/dart/server/internal/remote/utilities/RequestUtilities.java

```diff
@@ -145,25 +146,33 @@ public static JsonElement buildJsonElement(Object object) {
         }
       }
       return jsonObject;
-    } else if (object instanceof AnalysisError) {
-      return buildJsonObjectAnalysisError((AnalysisError) object);
-    } else if (object instanceof AddContentOverlay) {
-      return ((AddContentOverlay) object).toJson();
-    } else if (object instanceof ChangeContentOverlay) {
-      return ((ChangeContentOverlay) object).toJson();
-    } else if (object instanceof RemoveContentOverlay) {
-      return ((RemoveContentOverlay) object).toJson();
-    } else if (object instanceof AnalysisOptions) {
-      return ((AnalysisOptions) object).toJson();
-    } else if (object instanceof Location) {
-      return buildJsonObjectLocation((Location) object);
+    }
+    else if (object instanceof AnalysisError) {
+      return buildJsonObjectAnalysisError((AnalysisError)object);
+    }
+    else if (object instanceof AddContentOverlay) {
+      return ((AddContentOverlay)object).toJson();
+    }
+    else if (object instanceof ChangeContentOverlay) {
+      return ((ChangeContentOverlay)object).toJson();
+    }
+    else if (object instanceof RemoveContentOverlay) {
+      return ((RemoveContentOverlay)object).toJson();
+    }
+    else if (object instanceof AnalysisOptions) {
+      return ((AnalysisOptions)object).toJson();
+    }
+    else if (object instanceof Location) {
+      return buildJsonObjectLocation((Location)object);
+    } else if (object instanceof ImportedElements) {
+      return ((ImportedElements)object).toJson();
     }
     throw new IllegalArgumentException("Unable to convert to JSON: " + object);
   }
 
   /**
    * Generate and return a {@value #METHOD_ANALYSIS_GET_ERRORS} request.
-   * 
+   * <p>
```

Comment:
I thought `pre` forced a line break (making the `p` unnecessary), but perhaps I'm misremembering.